### PR TITLE
links contributions adjust

### PR DESCRIPTION
**Pull request by jessilyneh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,10 +119,17 @@ Abstract:
 
 
 _Tags: `translation`, `javascript`_
+<<<<<<< HEAD
```
Comment (ErickWendel) [inline README.md]: I think this shouldn't be here

File: resources/talks.json [hunk 2]
```diff
@@ -21,8 +21,13 @@
         "additionalLinks": [""],
         "language": "pt-br",
         "date": "2022-07-16"
+<<<<<<< HEAD
```
Comment (ErickWendel) [inline resources/talks.json]: all over the code oO

**Review by ErickWendel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jessilyneh) [reply README.md]: done

File: resources/talks.json (hunk 2 above)
Comment (jessilyneh) [reply resources/talks.json]: done

**Pull request merged**